src/services/boundaryService.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import config from '../../config/index.js'
+
+export const getBoundaryForLpa = async (boundaryId) => {
+  try {
+    const [datasetId, lpaId] = boundaryId.split(':')
+    if (!datasetId || !lpaId) {
+      return {
+        error: 'Invalid boundary ID'
+      }
+    }
+
+    const response = await axios.get(`${config.mainWebsiteUrl}/entity.json?dataset=${datasetId}&reference=${lpaId}`)
+    const entity = response?.data?.entities?.[0] ?? null
```

Comment:
That `... ?? null` bit isn't necessary - if anything on the left is missing you'll get `undefined ?? null` and nothing in the remaining code makes use of distinction between `null` and `undefined`.